### DISCUSSION TITLE
Trouble signing up for a  student account

### DISCUSSION THREAD

**Discussion by pfmephisto** [community/community#75869] [GitHub Education]
### Hi everyone,

Hi everyone ,
I'm a PhD at the Aarhus School of Architecture's PhD School it's a small PhD school that is part of the larger Aarhus School of Architecture.
As a PhD fellow we don't get student email addresses rather we are treated as equal to staff. I think this and the fact that all our enrollment documents are issued as digital PDF's is giving me trouble signing up for the GitHub Campus program. I have tried signing up as Student and as Teacher both without any luck. 

Is there another way for me to enroll in the GitHub Campus program? Or would it be possible to get the PhD school listed as separate school or enrolling as a teacher?

my last rejection email had the following reasons:

_•	Attendees of this school must verify and use their academic email address.
•	Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your last name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields.
•	Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy._

1. As previously mentioned as PhD students we don't get student email adresses, rather we have no extension and are using a staff email.
2. I have check my details under billing information and they appear correctly.
3. I only have a PDF of my enrolment letter. The resolution of my device camera is too low for the text to be clearly readable so I uploaded a photo of a print out that I took with my phone.

**Comment by AidenJamesYt** [comment]
As a workaround, consider creating a temporary email address that you can use specifically for the GitHub Campus program registration. This could be a Gmail or Outlook account that you can later abandon or forward to your staff email.

**Reply by pfmephisto** [reply]
Hi @AidenJamesYt  ,
thanks for your suggestion, but as I understand it GitHub requires me to use an email address that it associates with a university to participate in the campus program. The issue as I see it is that GitHub only has the email template for student email addresses and not the one that academic staff are using for my university.
eg. `[id]@stud.aarch.dk` (students) vs, `[initials]@aarch.dk` (PhD students and academic staff)
